core/backend/src/rpc-impl/RpcBriefcaseUtility.ts

```diff
@@ -131,6 +131,7 @@ export class RpcBriefcaseUtility {
     db = SnapshotDb.tryFindByKey(CheckpointManager.getKey(checkpoint));
     if (db) {
       Logger.logTrace(loggerCategory, "Checkpoint was already open", () => ({ ...tokenProps }));
+      BriefcaseManager.logUsage(requestContext, tokenProps);

```

Comment:
You'll need a lint ignore here for floating promise.

Reply:
Ignore me, it's in the logUsage call...

Reply:
// eslint-disable-line @typescript-eslint/no-floating-promises